lib/engine/route.rb

```diff
@@ -5,7 +5,7 @@
 module Engine
   class Route
     attr_accessor :halts, :routes
-    attr_reader :last_node, :phase, :train, :abilities
+    attr_reader :last_node, :phase, :train, :abilities, :local_length
```

Comment:
why does this need to be an attribute? no one uses it

Reply:
removed; I put there because originally I was logging it and ruby code was failing :)